gridcomps/History/Sampler/MAPL_TrajectoryMod_smod.F90

```diff
@@ -486,8 +504,7 @@
             do k=1, this%nobs_type
                if (this%obs(k)%nobs_epoch > 0) then
                   filename=trim(this%obs(k)%name)//trim(filename_suffix)
-                  call lgr%debug('%a %a', &
-                       "Sampling to new file : ",trim(filename))
+                  write(6,'(1x,a,2x,a)')  "Sampling to new file :",trim(filename)
```

Comment:
Is this printing on all PES/

Reply:
That is inside L503  if (mapl_am_I_root()) then.
The change was because @atrayano and others complain not seeing any output from sampler especially the trajectory sampler during NR. That also reveals writing netCDF on root takes a lot of time in this case. We need another PR for oserver.